docs/ODK_migrated/TermComments.md

```diff
@@ -0,0 +1,27 @@
+**NOTE** This documentation is incomplete, for now you may be better consulting the [GO Editor Docs](http://wiki.geneontology.org/index.php/Ontology_Editing_Guide)
```

Comment:
No need to put these files into "ODK_migrated" - just put them were they belong in your opinion (I know maybe not the time yet :P)

Reply:
honestly, just a little bit easier for me to keep track - if its not too big a deal, can we leave it that way in the folder structure? the front facing will end up the same with the index - if not happy to change after.

Reply:
moved